DM-2111 - grafted changelog to y2024 (#2391)
SoftwareAG/c8y-docs: content/change-logs/device-management/lwm2m-agent-10-18-54-196-possible-to-disable-required-availability-mgmt.md

---

> date:

Reviewer: ```suggestion
date: 2024-09-03
```

title: Disabling automatic setting of required interval of LWM2M agent
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
product_area: Device Management & Connectivity
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:

>   - value: tc-ggH2M4hf3
>     label: lwm2m-agent

Reviewer: ```suggestion
  - value: tc-QHwMfWtBk7
    label: cumulocity
```

ticket: DM-2111

> version: 10.18.54.196

Reviewer: ```suggestion
version: 10.18.540.195
```

---

> LWM2M agent manages required interval of LWM2M device automatically based on LWM2M registration lifetime. It's now possible to disable this behavior globally or for specific devices. 

Reviewer: There is also user documentation part in the develop PR
https://github.com/SoftwareAG/c8y-docs/pull/2305/files

Reviewer 2: I assume that the user documentation changes need to be grafted as well. Will this happen here or in a separate PR?

Author: missed those - done!

> LWM2M agent manages required interval of LWM2M device automatically based on LWM2M registration lifetime. It's now possible to disable this behavior globally or for specific devices. 

Reviewer 2: ```suggestion
The LWM2M agent automatically manages the required interval of a LWM2M device based on the LWM2M registration lifetime. It is now possible to disable this behavior globally or for specific devices. 
```

> The LWM2M agent automatically manages the required interval of a LWM2M device based on the LWM2M registration lifetime. It is now possible to disable this behavior globally or for specific devices which will result in a standard Cumulocity connection monitoring as described [in the documentation](https://cumulocity.com/docs/device-management-application/monitoring-and-controlling-devices/#to-monitor-the-connection-of-a-particular-device). 

Reviewer 2: ```suggestion
The LWM2M agent automatically manages the required interval of a LWM2M device based on the LWM2M registration lifetime. It is now possible to disable this behavior globally or for specific devices which results in a standard {{< product-c8y-iot >}} connection monitoring as described in [Monitoring and controlling devices](https://cumulocity.com/docs/device-management-application/monitoring-and-controlling-devices/#to-monitor-the-connection-of-a-particular-device). 
```
